### DISCUSSION TITLE
Yeelight FlexSwitch function and Tasmota

### DISCUSSION THREAD

**Discussion by MyHoTr** [arendst/Tasmota#21268] [Support request]
Friends, there are several Yeelight Ceiling Light (ESP-WROOM-32D) with support for the Yeelight FlexSwitch function. I successfully flashed to Tasmota. everything works with HASSIO. However, I cannot configure the slisaON function. Do you have any ideas how to do this?

**Comment by sfromis** [comment]
What do you expect to get with that "FlexSwitch"?

The point is that flashing the ceiling light with Tasmota totally replaces existing firmware and features. However, you may be able to do something similar with Tasmota.

**Reply by MyHoTr** [reply]
it is necessary to switch the device during a short interruption of electricity (with a special switch or relay, as in my case). thus, it always remained with the supply of electricity. even when the LEDs are not lit.

**Reply by sfromis** [reply]
Something similar was done on Sonoff Basic R4. https://github.com/arendst/Tasmota/pull/20247 https://github.com/arendst/Tasmota/discussions/20067

**Reply by MyHoTr** [reply]
Thanks for the idea

**Comment by MyHoTr** [comment]
Obviously one of the gpio needs to be monitored and if there is a momentary change from 220 volts to 0 volts the software switch is triggered. I can't find gpio that changes this state in real time.
